test/audit/models/ChargeSummaryAuditSpec.scala

```diff
@@ -249,6 +277,156 @@ class ChargeSummaryAuditSpec extends WordSpecLike with MustMatchers {
           )
         }
 
+        /*"there are charge details with coding out accepted" in {
```

Comment:
looks like one unit tests is cemented out, I would assume we need it?

Reply:
done